{bio}[intel/2018b] Pychopper v0.4.0

HPC-UniOldenburg opened the pull request:
(created using `eb --new-pr`)

verdurin commented on easybuild/easyconfigs/p/Pychopper/Pychopper-0.4.0-intel-2018a-Python-3.7.0.eb:
```diff
@@ -0,0 +1,47 @@
+]
+
+dependencies = [
+    ('Python', '3.7.0'),
```
@HPC-UniOldenburg - this version of `Python` is not available for the `intel/2018a` toolchain. It would be best either to use a different toolchain, or to use one of the `Python` versions that is available.

HPC-UniOldenburg commented:
@verdurin 
I uploaded a new version with fitting toolchain and changed the filename accordingly and deleted the old file.
Nevertheless, the check finds an error with the toolchain I formerly chose. Did I do something wrong?

smoors commented on easybuild/easyconfigs/p/Pychopper/Pychopper-0.4.0-intel-2018b-Python-3.7.0.eb:
```diff
@@ -0,0 +1,47 @@
+
+name = 'Pychopper'
+version = '0.4.0'
+versionsuffix = '-Python-3.7.0'
```
@HPC-UniOldenburg the 2018b toolchain generation requires Python-3.6.6 when used as a dependency

boegel commented:
@HPC-UniOldenburg An updated easyconfig for `Pychopper` is available in #10344.

Do you think it's worth cleaning up this easyconfig accordingly, or shall we close this PR?

boegel commented:
Closing this in favor of #10344, since `intel/2018b` toolchain is [deprecated](https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains)
